integration/spark-common-test/src/test/scala/org/apache/carbondata/spark/testsuite/localdictionary/LocalDictionarySupportCreateTableTest.scala

```diff
@@ -1575,7 +1571,7 @@ class LocalDictionarySupportCreateTableTest extends QueryTest with BeforeAndAfte
     val descLoc = sql("describe formatted local1").collect
 
     descLoc.find(_.get(0).toString.contains("Local Dictionary Enabled")) match {
-      case Some(row) => assert(row.get(1).toString.contains("true"))
+      case Some(row) => assert(row.get(1).toString.contains("false"))
```

Comment:
If local dictionary is disabled here, it will be the same as previous test case. Can you make it true and add other properties like local_dictionary_threshold or local_dictionary_include, etc.

Reply:
akashrn5 just now  Contributor
this scenario is already covered